cli/cli/kurtosis_config/resolved_config/kurtosis_cluster_config.go

```diff
@@ -86,29 +78,29 @@ func (clusterConfig *KurtosisClusterConfig) GetClusterType() KurtosisClusterType
 func getSuppliers(clusterId string, clusterType KurtosisClusterType, kubernetesConfig *v2.KubernetesClusterConfigV2) (
 	kurtosisBackendSupplier,
 	engine_server_launcher.KurtosisBackendConfigSupplier,
-	*engine_server_launcher.KurtosisRemoteBackendConfigSupplier,
 	error,
 ) {
-	kurtosisRemoteBackendConfigSupplier := engine_server_launcher.NewKurtosisRemoteBackendConfigSupplier(store.GetContextsConfigStore().GetCurrentContext)
-
 	var backendSupplier kurtosisBackendSupplier
 	var engineConfigSupplier engine_server_launcher.KurtosisBackendConfigSupplier
 	switch clusterType {
 	case KurtosisClusterType_Docker:
 		if kubernetesConfig != nil {
-			return nil, nil, nil, stacktrace.NewError(
+			return nil, nil, stacktrace.NewError(
 				"Cluster '%v' defines cluster config, but config must not be provided when cluster type is '%v'",
 				clusterId,
 				clusterType.String(),
 			)
 		}
 
 		backendSupplier = func(_ context.Context) (backend_interface.KurtosisBackend, error) {
-			kurtosisRemoteBackendConfigMaybe, err := kurtosisRemoteBackendConfigSupplier.GetOptionalRemoteConfig()
-			if err != nil {
-				return nil, stacktrace.Propagate(err, "Error building optional remote Kurtosis backend config")
+			var remoteBackendConfigMaybe *configs.KurtosisRemoteBackendConfig
+			currentContext, _ := store.GetContextsConfigStore().GetCurrentContext()
+			if currentContext != nil {
```

Comment:
Interesting pattern here, but I think go generally prefers doing something like:
```
currentContext, err := store.GetContextsConfigStore().GetCurrentContext()
if err != nil {
    handle the error
} else {
    // you can assume the result object (here `currentContext `) is not nil. 
}
```
And if the first `if err != nil` returns, the `else` is not necessary and you can un-indent the case of `currentContext != nil`

Reply:
I had copied from somewhere else.  I agree with you that the pattern lacked consistency with the rest of the code and I changed it.